### ISSUE TITLE
[docs] Code blocks don't render on Firefox mobile browser

### ISSUE THREAD

**Issue by achou11** [modernweb-dev/rocket#340]
I'm using Firefox Nightly on Android (12) and some of the code blocks don't render properly. The text is definitely there so I guess it's some CSS issue? I tried on a Chromium-based mobile browser (Vivaldi) and that worked fine.

Example:

![Screenshot_20220324-100842](https://user-images.githubusercontent.com/18542095/159935000-25cb64bd-2936-47b1-b6f9-2b962f214078.png)

**Comment by houseofmercy-github** [comment]
Seeing the same behavior on 99.0.1 (64-bit) OSX desktop.

Suspect the problem is in the `<code-tabs>` because the `<pre class="language-bash">` containing the actual text appears fine on other pages and manually moving those elements out of the `<code-tabs>` with the inspector causes the text to appear.

**Comment by daKmoR** [comment]
new rocket version introduced quite a change... I don't think that is an issue anymore - if it is feel free to reopen or creating a new issue 🤗

**Comment by houseofmercy-github** [comment]
Just did a quick check to see if fixed but now things are very bad.  Nothing appears when I visit the site.
Looking at browser console shows an error.

![rocket](https://user-images.githubusercontent.com/103275384/184506486-41db3593-6b1b-4bc7-a8ea-4e58e55cefa5.png)

**Comment by daKmoR** [comment]
hmmm - strange 🤔

I just tested it on Firefox 103.0.2 (64-bit) OSX...

what you showed seems like an old service worker? 🤔
can you try a "hard" reload - typically that is `Command + Shift + R`? 🤞🙏

**Comment by houseofmercy-github** [comment]
Updated to Firefox 103.0.2 and tried a hard reload but it didn't change the outcome.
But I do see a slightly different backtrace.   Here is the "Save all messages to File" from the console.

[console-export-2022-8-13_13-24-55.txt](https://github.com/modernweb-dev/rocket/files/9332120/console-export-2022-8-13_13-24-55.txt)

**Comment by daKmoR** [comment]
hmmm as I could not reproduce it I asked 4 different people to try it and they all said it worked fine on their FF... 3 with mac - 1 with windows....

so I'm at a loss here 🙈 

some ideas
- open it via a private tab
- use Dev Tools (F12) -> Application Tab -> [unregister] (Service Worker)
- open settings (about:preferences#privacy) => Privacy & Security => Cookies and Site Data => Clear Data... => select ONLY "Cached Web Content" => clear

**Comment by houseofmercy-github** [comment]
None of those suggestions helped, but disabling all Firefox add-ons fixed the issue.
After carefully re-enabling them one by one I found https://rocket.modern-web.dev 
will error as decribed earlier if **either** of these extensions is enabled:

* **Stylus** https://addons.mozilla.org/en-US/firefox/addon/styl-us
* **Disable HTML5 Autoplay** https://addons.mozilla.org/en-US/firefox/addon/disable-autoplay

After re-enabling my other extensions but leaving those disabled I was able to view the site.
I don't know what those extensions are doing that would cause the error.  I've found that

* enabling **Stylus** and configuring it to ignore the modern-web.dev domain works ✅
* enabling **Disable HTML5 Autoplay** breaks the site even if configured to ignore the domain❗️

so I think I'll have to make a note of this in the future.

Getting back to the original issue - the docs look different from the ones in April and 
I don't see any missing code sections like I did before.  I couldn't find any obvious "tabbed" 
code examples so either they aren't used anymore or are working as expected.
